app/services/foreman/plugin.rb

```diff
@@ -67,6 +67,16 @@ def register(id, &block)
         registered_plugins[id] = plugin
       end
 
+
+      def modify_backtrace_cleaner
+        cleaner = Rails.backtrace_cleaner
+        cleaner.remove_silencers!
+        cleaner.add_silencer do |line|
+          line !~ Rails::BacktraceCleaner::APP_DIRS_PATTERN &&
+              !(all.any? { |plugin| line.include?(plugin.name) })
```

Comment:
Should we replace the filter?  Since line.include?(plugin.name) will only likely work when plugins are namespaced, which isn't all that many - and usually, not every file.